source/lib/financials/balances.js

```diff
@@ -6,14 +6,21 @@ import * as cache from '../cache'
 import {FINANCIALS_URL, LANDING_URL} from './urls'
 import type {FinancialDataShapeType} from './types'
 
-type PromisedDataType = Promise<{error: true, value: Error}|{error: false, value: FinancialDataShapeType}>;
-
-export async function getFinancialData(isConnected: boolean, force?: bool): PromisedDataType {
+type PromisedDataType = Promise<
+  | {error: true, value: Error}
+  | {error: false, value: FinancialDataShapeType}>;
```

Comment:
I'm going to file an issue about this chunk; I don't like how it breaks the contents after the `<` but leaves the closing `>` on the closing line.